### PR TITLE
Switch to golangci-lint

### PR DESCRIPTION
**Pull request by jbowes**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: go
 go:
-- 1.11.x
+- "1.11.x"
+- "1.12.x"
+- "1.13.x"
```
Comment (clwiseman) [inline .travis.yml]: I'm just curious: why three go versions?

**Review by clwiseman — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (jbowes) [reply .travis.yml]: I didn't want to drop 1.11 if I didn't have to, even though it's no longer supported.

Then, I added in the supported versions :) In theory if it works on an older version it should work on the newer ones, but in practice you never know!

Comment (jbowes) [reply .travis.yml]: and 1.11 didn't work, so out it goes!

Comment (clwiseman) [reply .travis.yml]: Makes sense! Thanks!

**Pull request merged**